### DISCUSSION TITLE
Can you please a try on this pkg and how to do it ?

### DISCUSSION THREAD

**Discussion by Rishabh-creator601** [yangheng95/PyABSA#224] [Ideas]
The project is not on basis on simple aspects but some keywords

dataset Type :  ratings and reviews of app store 

statements are like this : 
=> For example if a user says this app runs easy as smooth.. that is related to usability
=> For example if they say this app is great , runs smoothly that relates to usability
=> But if they say the app is not worth paying for That’s pricing issue
=> At the end , I want to know the percentage of sentiments that mostly bother on usability. Compared to other aspects.
=> The key aspects I’m looking at 6 aspects General, Reliability, Usability, Functionality, Pricing, Support

If you want to do it please let me know it is a task given by my one mate and i dont know how do it with your package

**Comment by yangheng95** [comment]
I am sorry your work is asepct category detection that can not use this tool

**Reply by Rishabh-creator601** [reply]
ok Thnx
